diff --git a/doc/htmldoc/conf.py b/doc/htmldoc/conf.py
index 2206227a4..66230fda2 100644
--- a/doc/htmldoc/conf.py
+++ b/doc/htmldoc/conf.py
@@ -92,7 +92,7 @@ copybutton_prompt_text = ">>> "
 copybutton_only_copy_prompt_lines = True

 mermaid_output_format = "raw"
-mermaid_version = "10.2.0"
+mermaid_version = "10.3.0"

 # disable require js - mermaid doesn't work if require.js is loaded before it
 nbsphinx_requirejs_path = ""
